OpenOversight/app/main/forms.py

```diff
@@ -514,8 +514,8 @@ class IncidentForm(DateFieldForm):
     report_number = StringField(
         validators=[
             Regexp(
-                r"^[a-zA-Z0-9-]*$",
-                message="Report numbers can contain letters, numbers, and dashes",
+                r"^[a-zA-Z0-9- ]*$",
+                message="Report cannot contain special characters (dashes permitted)",
             )
         ],
```

Comment:
Does this `report_number` correspond to the OPA case number? I'm a bit confused by the space in the regex

Reply:
It used to, but we specifically had a request from a user to have all OPA case numbers prefixed with the text "OPA Case". Incidents is already pretty generic, so I felt it made sense to allow spaces in the report number

Reply:
Makes sense, thank you for clarifying!